src/main/java/com/google/sps/servlets/DashboardServlet.java

```diff
@@ -33,14 +33,15 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-    String startTimestamp = RequestUtils.getParameter(request, Property.TIMESTAMP);
-    String endTimestamp;
+    String startTimestamp = RequestUtils.getParameter(request, "startTime");
+    String endTimestamp = RequestUtils.getParameter(request, "endTime");
 
-    if (startTimestamp.isEmpty()) {
+    // endTimestamp should include the entire second date requested
```

Comment:
what do you mean by entire "second date"

Reply:
when they give a range of dates, i include the latter date as well.

Reply:
okay so just change the comment to make that clearer "When displaying the range of dates, the latter date is included."